### ISSUE TITLE
Introduce a new type of AsciiDoc table cell

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor#2352]
Currently, an AsciiDoc table cell works like an embedded document. While this is useful for some cases, it's often too much when all you want to do is use block-level AsciiDoc content in a table cell.

The proposal here is to split the AsciiDoc table cell style into two styles, `a` and `A`. The `a` style allows block-level content, but otherwise acts like any other delimited block. The `A` style allows a full AsciiDoc document inside, and converts it as self-contained. In other words, it has it's own document header, it's own table of contents, and it's own footnotes (which is currently how the AsciiDoc table style works).

In summary:

* `a` - acts similar to a delimited block (sidebar, example, etc)
* `A` - acts as an embedded document

Of course, we're going to have a problem with compatibility, so we may have to invert these two values (even thought that's not as intuitive). Another option is to use `b`, which stands for "block". That might be a reasonable trade-off.

Related issues: #1705, #2350.

**Comment by 71GA** [comment]
Has this already been implemented? It is now 2025.

**Comment by mojavelinux** [comment]
No. This is a change that would have to happen in the AsciiDoc Language project. However, it's not something we are currently discussing, so it would likely be something that is considered after the 1.0 version of the specification. Therefore, it would be at least a year out if not further.
